documentation/asciidoc/topics/ref_batch_operations.adoc

```diff
@@ -0,0 +1,105 @@
+[id='batch-operations_{context}']
+= Example Batch Operations
+
+Use these example batch operations as starting points for creating and modifying {brandname} resources with the `Batch` CR.
+
+== Users
+
+Create several {brandname} users and assign them roles with varying levels of permission to access caches and interact with {brandname} resources.
+
+[source,sh,options="nowrap",subs=attributes+]
+----
+echo "connecting"
+connect --username=admin --password=changeme
+echo "creating users and groups..."
+create user katie -p changeme -g deployer
+user create --groups=developers developers -p changeme
+user roles grant --roles=application developers
+create user john -p changeme -g developers
+create user dave -p changeme -g developers
+create user mark -p changeme -g developers
+create user julia -p changeme -g developers
+create user pedro -p changeme -g developers
+create user rita -p changeme -g observer
+echo "list users"
+user ls
+----
+
+== Caches
+
+Create multiple caches from configuration files.
+
+[NOTE]
+====
+You can pass configuration files to {ispn_operator} only via a `ConfigMap`.
+====
+
+[source,sh,options="nowrap",subs=attributes+]
+----
+echo "connecting"
+connect --username=admin --password=changeme
+echo "creating caches..."
+create cache sessions --file=infinispan-prod-sessions.xml
```

Comment:
The configmap files are all mounted at `/etc/batch` in the pod that's executing the batch, so it's actually necessary to prepend the file with this dir, e.g. `--file=/etc/batch/infinispan-prod-sessions.xml`.